### PR TITLE
First Pass at Compiling List

### PR DESCRIPTION
**Pull request by earthtone**
- not sure if list is too long/overwhelming
- not sure if some of the material is more advanced than necessary
- not sure if i missed/overlooked something obvious

### CODE REVIEW

File: resources.md [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+- [Professor Frisby's Mostly Adequate Guide to Functional Programming in JavaScript](http://funpress.github.io/mostly-adequate-guide-to-fp) - In-depth survey of functional programming concepts
+
+### Video Courses (Free & Paid)
+
+- [Node Tuts](http://nodetuts.com/) - Video tutorials about core Node JS concepts
+- [FunFunFunction](https://www.youtube.com/channel/UCO1cgjhGzsSYb1rsB4bFe4Q) - Technical concepts presented in a simple, non-technical format
+- [Egghead](https://egghead.io/) - Bite-sized and fast-paced courses on popular web technologies
+- [Front End Masters](https://frontendmasters.com/) - Classroom style workshops on specific web technologies ( not just the *Front End* )
+- [Wes Bos](https://wesbos.com/courses/) - In-depth, step-by-step courses on specific technologies
+- [Classroom Coding with Professor Frisby](https://www.youtube.com/playlist?list=PLK_hdtAJ4KqX0JOs) - Stop motion animation course on the fundamental concepts of Funcitonal Programming
```
Comment (vitalbone) [inline resources.md:72]:
```suggestion
- [Classroom Coding with Professor Frisby](https://www.youtube.com/playlist?list=PLK_hdtAJ4KqX0JOs) - Stop motion animation course on the fundamental concepts of functional programming
```

File: resources.md [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+
+### Coding Environments
+
+- [Code Sandbox](https://codesandbox.io/)
+- [Codepen](https://codepen.io/)
+- [Glitch](https://glitch.com/)
+- [JS Fiddle](https://jsfiddle.net/user/login/)
+- [JS Bin](https://jsbin.com/?html,js,output)
+- [Require Bin](http://requirebin.com/)
+- [Observable](https://observablehq.com/)
```
Comment (vitalbone) [inline resources.md:87]:
```suggestion
- [Observable](https://observablehq.com/)
- [Repl.it](https://repl.it/languages/javascript)
```

**Review by vitalbone — APPROVED** [review]
Amazing. Thanks heaps @earthtone 

Would you mind if we eventually moved this to the wiki so we can break it up a little to make it easier to digest for beginners?

**Comment by earthtone** [comment]
@vitalbone I'm happy for this to sit in the wiki rather than as a raw MD file. I was mostly inspired by all those `Awesome X` lists people compile, since I used those a lot when trying to learn something new, but as I started to organize it I definitely noticed some of the stuff I added was more geared towards beginners while other resources presume some base understanding of things like JavaScript or web development in general. Hence my question in the commit message as to whether a massive list like this is the best approach.

**Comment by vitalbone** [comment]
Cool. I'll merge this in and then move things around a little.

**Pull request merged**